dev/CommonStyles/MenuFlyout_themeresources.xaml

```diff
@@ -508,6 +516,10 @@
                                                 Storyboard.TargetProperty="Foreground">
                                             <DiscreteObjectKeyFrame KeyTime="0" Value="{ThemeResource MenuFlyoutSubItemForegroundDisabled}" />
                                         </ObjectAnimationUsingKeyFrames>
+                                        <ObjectAnimationUsingKeyFrames Storyboard.TargetName="Icon"
```

Comment:
I think this should be IconContent also?

Reply:
Good catch, that was a typo.